inventory_store_mysql/service.bal

```diff
@@ -0,0 +1,37 @@
+import ballerina/http;
+import ballerinax/mysql;
+import ballerina/sql;
+import ballerinax/mysql.driver as _;
+
+listener http:Listener helloEP = new(9090);
+
+type Item record {
+    int id?;
+    string name;
+    int quantity;
+};
+
+configurable string host = ?;
+configurable int port = ?;
+configurable string user = ?;
+configurable string password = ?;
+configurable string database = ?;
```

Comment:
Can't we make a configuration record and use it ? Check the following code sample.
- https://github.com/shafreenAnfar/ballerina-social-media/blob/72187dd41e0e40f4e779534cabdb43ef6fb9068f/social-media-service/social_media_service.bal#L14